cli/src/asset_identifier.rs

```diff
@@ -0,0 +1,66 @@
+use error::InvalidInputError;
+use stellar_client::resources::AssetIdentifier;
+
+pub fn parse_identifier_from_str(s: &str) -> Result<AssetIdentifier, InvalidInputError> {
+    let asset_parts: Vec<&str> = s.split("-").collect();
+    if asset_parts[0] == "xlm" {
+        return Ok(AssetIdentifier::Native);
+    }
+    if asset_parts.len() != 2 {
+        return Err(InvalidInputError {
+            details: "Asset identifier not of the form <asset_code>-<asset_issuer>".to_string(),
+        });
+    }
+    if asset_parts[0].len() < 5 {
+        return AssetIdentifier::new(
```

Comment:
I think there are easier to use constructors than this now:

https://github.com/kbacha/stellar-sdk/blob/master/client/src/resources/asset.rs#L145-L163